weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectsPage/ObjectVersionsTable.tsx

```diff
@@ -83,6 +82,12 @@ export const ObjectVersionsTable: React.FC<{
         // solution here in the future. Maybe exclude table refs?
         val = _.omit(val, 'rows');
       }
+      // This is a dirty hack to enable keeping the nested query value for Monitors
```

Comment:
I had to really think hard about what this hack is doing. Do you mind adding a comment here describing what is going on? In particular, i think the summary is: `keepNestedVal` allows you to specify top-level fields from the underlying vals which are not flattened into nested columns.